### PR TITLE
Added XML comments for the repositoryUpdateConductor methods

### PR DESCRIPTION
**Pull request by SaidShah**
- [x] Related GitHub issue(s) linked in PR description
- [x] Destination branch merged, built and tested with your changes
- [x] Code formatted and follows best practices and patterns
- [x] Code builds cleanly (no _additional_ warnings or errors)
- [x] Manually tested
- [x] Automated tests are passing
- [x] No _decreases_ in automated test coverage
- [x] Documentation updated (readme, docs, comments, etc.)
- [x] Localization: No hard-coded error messages in code files (_minimally_ in string constants)

#12

### CODE REVIEW

File: src/AndcultureCode.CSharp.Conductors/RepositoryUpdateConductor.cs [hunk 1]
```diff
@@ -19,37 +26,59 @@ public class RepositoryUpdateConductor<T> : Conductor, IRepositoryUpdateConducto
...
+        /// <param name="items">List of items to update</param>
+        /// <param name="updatedBy">Id of user updating the entity</param>
+        /// <returns></returns>
         public virtual IResult<bool> Update(IEnumerable<T> items, long? updatedBy = default(long?))
         {
             return _repository.Update(items, updatedBy);
         }
 
-        #endregion
+        #endregion IRepositoryUpdateConductor Implementation
```
Comment (wintondeshong) [inline src/AndcultureCode.CSharp.Conductors/RepositoryUpdateConductor.cs:82]: Nitpick: Mind removing this extra newline?

**Review by wintondeshong — CHANGES_REQUESTED** [review]
@SaidShah aside from the minor removal of the newline, good to go. Let us know when you've made the change and we will merge.

**Comment by SaidShah** [comment]
@wintondeshong thanks, it should be good to go now.

**Review by wintondeshong — APPROVED** [review]
looks good! Thanks @saidshahdev

**Comment by brandongregoryscott** [comment]
@all-contributors add @SaidShah for doc

**Pull request merged**